src/mscorlib/shared/System/Text/StringBuilder.cs

```diff
@@ -1033,6 +1039,21 @@ public StringBuilder Append(char[] value)
             return this;
         }
 
+        public StringBuilder Append(ReadOnlySpan<char> value)
+        {
+            if (value.Length > 0)
+            {
+                unsafe
+                {
+                    fixed (char* valueChars = &value.DangerousGetPinnableReference())
```

Comment:
Calls `string.wstrcpy` though which wants pointers; would need to use equivalent span function in string to that?